content/load-testing/optimization-and-tuning-procedures.md

```diff
@@ -6,13 +6,13 @@ weight: 4
 
 Here are some optimization and tuning steps that we usually perform after running load tests:
 
-* Put all static assets behind a CDN (e.g. AWS CloudFront), do not overload the Kubernetes pods with serving the static assets (in may cases, this is one of the main reasons of poor website performance)
-* Scale Kubernetes cluster (horizontally by adding nodes or vertically by using different types of EC2 instances)
-* Scale Kubernetes pods (horizontally by increasing the replica count)
-* Scale pods Memory
-* Scale pods CPU
-* Scale Nginx Ingress pods (horizontally by increasing the replica count)
-* Scale Nginx Ingress CPU and Memory
-* Tune Nginx paramaters (e.g timeouts, queues)
-* Optimize application/web server parameters (e.g. concurrency, threads and processes, thread pools, timeouts, memory limits)
-* Optimize database queries and indexes
+* Put all static assets behind a CDN (e.g. AWS CloudFront), do not overload the Kubernetes pods with serving the static assets (in many cases, this is one of the main reasons of poor website performance)
+* [Scale Kubernetes cluster horizontally by adding nodes]({{< relref "scaling-kubernetes-cluster/scale-cluster-horizontally.md" >}})
+* [Scale Kubernetes cluster vertically by using different types of EC2 instances]({{< relref "scaling-kubernetes-cluster/scale-cluster-vertically.md" >}})
```

Comment:
I don't like fully linked sentences. Link 2-3 words max. Generally use a consistent convention like this:

```
* [Scale Verticall](...) - Scale cluster vertically by using different....
* [Scale Horizontally](...) - ...
* [Scale Ingress](...) - ...
```


`